docs/services/sensors.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Sensors Service"
+linkTitle: "Sensors"
+weight: 70
+type: "docs"
+description: "The Sensors Service is a built-in service that provides a central interface to all of your robot's sensors."
```

Comment:
```suggestion
description: "The Sensors Service provides a central interface to all of your robot's sensors."
```
Maybe not necessary to specify built-in? Or is this in anticipation of the addition of non-built-in services? In either case maybe the description can be more concise this way and the first paragraph can have the full version if you want to leave it in.

Reply:
Also consider changing "to" to "for"?